frontend/src/components/teacher/student-management/classrooms/AddClassroomModal.tsx

```diff
@@ -39,6 +40,14 @@ type AddClassroomModalProps = {
   isOpen: boolean;
 };
 
+const isSameDay = (date1: Date, date2: Date): boolean => {
```

Comment:
nit: since we're adding the `date-fns` library already you can probably just use https://date-fns.org/v2.30.0/docs/isSameDay

Reply:
Ahh good catch!